### ISSUE TITLE
accounts-google not working for local host

### ISSUE THREAD

**Issue by pavlism** [meteor/guide#490]
When you try to configure you google login step 6 says: "Set Authorized Redirect URI to: http://localhost:3000 /_oauth/google"

But when I add "http://localhost:3000 /_oauth/google" to the Authorized redirect URIs, I get the following error message: Invalid Redirect: http://localhost:3000 /_oauth/google must end with a public top-level domain (such as .com or .org)

**Comment by batjko** [comment]
[StackOverflow says](http://stackoverflow.com/questions/26183980/possible-to-test-google-social-login-locally) you should create a local alias in your `/etc/hosts` file.

Although [someone](http://stackoverflow.com/a/36109854/1688159) was apparently to just use the localhost IP instead (`127.0.0.1`).

**Comment by pavlism** [comment]
I also got it working eventually, but the package should change it's instructions so people don't have to go searching for answers.

**Comment by lorensr** [comment]
Thanks for reporting @pavlism. How did you wind up getting it working?

> "Set Authorized Redirect URI to: http://localhost:3000 /_oauth/google"

What/where is this said? Eg client JS console?

**Comment by abernix** [comment]
I'm not sure this is still a problem; I tested Google recently during some `accounts-*` packages changes and it appeared to work fine with `localhost:3000`.

Closing as this hasn't seen a lot of activity lately but in the case that there _is_ a limitation, the best suggestion would be to use a localhost redirector service, such as [Forward](https://forwardhq.com/) (there are others, but I can't remember them at the moment!).  Happy to reopen if this continues to be a problem, but there haven't been any +1s in quite some time.  Thanks for reporting this originally though!
